atomic_reactor/inner.py

```diff
@@ -291,6 +291,8 @@ class ImageBuildWorkflowData(ISerializer):
     plugins_errors: Dict[str, str] = field(default_factory=dict)
     build_canceled: bool = False
     plugin_failed: bool = False
+    # build start time
+    start_time: float = field(default=time.time())
```

Comment:
Does this mean the start_time of the build will be the time when `inner.py` was first imported? That seems questionable, maybe we could still import the start_time from `atomic_reactor` and set that as the default?

Reply:
Yes, I also looked into setting the start time to pipeline start time but we don't get any pipeline metadata in our env.

Reply:
why not add it to DockerBuildWorkflow ? which is really used at the start

Reply:
I don't think it will persist through task runs.

Reply:
that would require passing it from osbs-client. I don't think that will be any better and adding someway in here to somehow pass `start_time` doesn't seem like a good idea.

Reply:
I didn't mean to add it to workflow, but just use start_time from prun, where ever you need directly

Reply:
updated to get start time from pipeline run